packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -35,6 +35,12 @@ const propTypes = {
    * Allows parent to toggle the component. True for open and false for close.
    */
   isOpen: PropTypes.bool,
+  /**
+   * Ref to the first hidden child element. The element will get focus once the full content revealed. That allows the assistive technologies to start reading the hidden content from the place where it was cut off.
+   */
```

Comment:
can we reword slightly:
```suggestion
  /**
   * Ref to the first hidden child element that will receive focus when the full content is revealed. This allows assistive technologies to start reading the hidden content from where it left off.
   */
```

Reply:
Good catch! Thank you for suggesting. Added here: https://github.com/cerner/terra-core/pull/3838/commits/848c29f87e0af5b43f6eaade4494fd8a82d0ba6e